README.md

```diff
@@ -158,6 +162,14 @@ In the terminals we've [benchmarked](http://github.com/jwilm/vtebench),
 Alacritty is either faster or **way** faster than the others. If you've found a
 case where this isn't true, please report a bug.
 
+**_Why isn't feature X implemented?_**
+
+Alacritty has many great features, but not every feature from every other
+terminal. This could be for a number of reasons, but sometimes it's just
+because we aren't interested in the added complexity. This means you won't find
+things like tabs or splits (which are best left to a window manager or
+[terminal multiplexer][tmux]) nor niceties like a GUI config editor.
```

Comment:
`we aren't interested in the added complexity` feels a bit dismissive I think. Putting it in a better light might be a good idea.

Something like this is a bit marketingy and kinda downplays the issue, but I think it might be more welcoming:

```
but sometimes it's just because the feature isn't a good fit for Alacritty.
```